python/py_vapid/utils.py

```diff
@@ -26,11 +26,14 @@ def b64urlencode(data):
     return base64.urlsafe_b64encode(data).replace(b'=', b'').decode('utf8')
 
 
-def num_to_bytes(n):
+def num_to_bytes(n, l):
```

Comment:
Looks like flake8 complains about `l` as a variable name. Could you use something like `length` or `pad_to`?